lib/myFTClient.js

```diff
@@ -436,6 +436,27 @@ function _getNodesFollowingConcept(licenceId, conceptId, nodeType) {
 }
 
 /**
+ * Gets nodes that are following concepts/topics through KAT
+ * @param {String} licenceId -
+ * @param {String} conceptId -
+ * @param {String} nodeType - user|group
+ * @param {Object} params - Any additional parameters needed to filter the follow relationship.
+ * @returns {Promise} response -
+ * @private
+ */
+function _getNodesFollowingConceptByKat (licenceId, conceptId, nodeType, params) {
+  const operation = `myFTClient.getNodesFollowingConceptByKat`;
+  log.debug({operation, licenceId, conceptId, nodeType, params});
+
+  return _createAndTriggerScopedRequestKat('GET', myftConst.licenceNodeName, licenceId, myftConst.topicNodeName, conceptId, myftConst.followedRelName, nodeType, null, params)
```

Comment:
Personally, I find "followers" easier to read in the URL, because it comes after the concept. So it kind of reads "concept 12345's followers".

I'll hard code it to "followers" for now.